apps/participants/views.py

```diff
@@ -108,3 +114,20 @@ def get_context_data(self, *args, **kwargs):
         context["participant_data"] = json.dumps(data, indent=4)
         context["participant_schedules"] = participant.get_schedules_for_experiment(experiment, as_dict=True)
         return context
+
+
+@login_and_team_required
```

Comment:
Is there a specific reason why you didn't go for a class based view here? That way these two views could be targeted by a single URI and be packaged together.

Reply:
I generally prefer function based views as they are simpler to with less hidden functionality. But I get your point about combining them.

Reply:
Up to you

Reply:
[c7cb2d2](https://github.com/dimagi/open-chat-studio/pull/677/commits/c7cb2d25520d03230ea21b86187c61de88a2aff9)